### PR TITLE
Update Go to 1.19.2

### PR DESCRIPTION
**Pull request by jakule**

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -3823,7 +3823,7 @@ steps:
   - cd $WORKSPACE_DIR/go/src/github.com/gravitational/teleport
   - build.assets/build-fido2-macos.sh build
   - export PKG_CONFIG_PATH="$(build.assets/build-fido2-macos.sh pkg_config_path)"
-  - make clean release OS=$OS ARCH=$ARCH FIDO2=yes TOUCHID=yes LIBPCSCLITE=yes
+  - make clean release OS=$OS ARCH=$ARCH FIDO2=yes TOUCHID=yes
```
Comment (jakule) [inline .drone.yml:3826]: Yes, this is "intentional" https://github.com/gravitational/teleport/pull/16986#pullrequestreview-1130255822

**Comment by codingllama** [comment]
@jakule, could you update teleport.e and change the e/ commit here too?

**Comment by jakule** [comment]
@codingllama Updated https://github.com/gravitational/teleport/pull/17020/commits/75960276d8e7e4590db4fac1bf65bca87e5e4861

**Review by codingllama — APPROVED** [review]
Thanks!

**Pull request merged**